release-notes/cloudflow-1.3.2.md

```diff
@@ -0,0 +1,40 @@
+# Cloudflow 1.3.2 Release Notes
+
+Cloudflow 1.3.2 contains the following changes:
+
+- Fixed a bug that was causing semi-random deletions of application resources after multiple subsequent deploys. This was caused by invalid instructions (so-called "owner references") for the Kubernetes garbage collector that were added to support clean undeploy/uninstall.
+- Fixed another side-effect of the above root cause, which could lead to broken Cloudflow installs on some platforms.
+- Fixed an issue with generated Kafka consumer IDs that could cause incorrect consumption in case a streamlet class was added to a blueprint multiple times. Kafka consumer IDs are once again globally unique.
+- Fixed an issue with the `kubectl cloudflow status` command not reporting the correct rolled up application status for applications that include Flink-based streamlets.
+- Fixed an issue with configuration of streamlet parameters during local running. Internal variable references inside the `local.conf` file were not resolved correctly.
```

Comment:
```suggestion
- Fixed an issue with the configuration of streamlet parameters during local running. Internal variable references inside the `local.conf` file was not resolved correctly.
```

Reply:
@olofwalker the subject is "references", which is plural so it should be "were".

Reply:
```suggestion
- Fixed an issue with the configuration of streamlet parameters during local running. Internal variable references inside the `local.conf` file were not resolved correctly.
```